modulo2/jsx-labetube/aula-labetube/src/App.js

```diff
@@ -0,0 +1,75 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  const titulo = "Titulo do video"
+  function reproduzVideo(); onClick={reproduzVideo}
+  {
+    alert("O vídeo está sendo reproduzido")
+
+  }
+ 
+  return (
+    <div>
+      <div className="tela-inteira">
+        <header>
+            <h1>Future Tube</h1>
+            <input type="text" placeholder="Busca" id="campoDeBusca" />
+        </header>
+
+        <main>
+            <nav className="menu-vertical">
+                <ul>
+                    <li className="botoes-meunu-vertical">Início</li>
+                    <li className="botoes-meunu-vertical">Em alta</li>
+                    <li className="botoes-meunu-vertical">Inscrições</li>
+                    <hr />
+                    <li className="botoes-meunu-vertical">Originais</li>
+                    <li className="botoes-meunu-vertical">Histórico</li>
+                </ul>
+            </nav>
+            
+            <section className="painel-de-videos">
+                <div className="box-pagina-principal media1" onclick="reproduzVideo()">
+                    <img src="https://picsum.photos/400/400?a=1 " alt="" />
+                    <h4>{titulo}</h4>
+                </div>
+                <div className="box-pagina-principal media2" onclick="reproduzVideo()">
+                    <img src="https://picsum.photos/400/400?a=2 " alt="" />
```

Comment:
E quando você for chamar essa função dentro do onclick, você precisa chamar a função como variável, outro detalhe é que no react a propriedade onclick vira onClick, ok? Segue o exemplo de como seria essas linhas:
~~~Js
<div className="box-pagina-principal media1" onClick={reproduzVideo}>
~~~